### ISSUE TITLE
2 main reason for not using this library!  

### ISSUE THREAD

**Issue by mhemon** [TheFinestArtist/FinestWebView-Android#111]
1) webview height,  when u open a url it should be fine, but when go to 2nd page it,s not change the webview page size.

2) webview dosen't go topword when u open a new url!

**Comment by mhemon** [comment]
it,s completely waste my lot,s of time :'(

**Comment by baoxiehao** [comment]
Yes, quite the same problem boring me... but it's a very good web view when ignoring this bug.

**Comment by ponnamkarthik** [comment]
> [https://github.com/PonnamKarthik/FinestWebView-Android](url)


Check the code from above link 
webview height issue is fixed
